### DISCUSSION TITLE
How to distribute child template?

### DISCUSSION THREAD

**Discussion by PhocaCz** [joomla/joomla-cms#40233] [Q&A]
Hi, 

let's say I have created a child template and now I would like to distribute it to users. So that they have it as easy as possible. 

Is there any installer package system for child templates? E.g. one that would create a child template during installation (if it doesn't already exist) and copy e.g. the overrides to the appropriate places?

If not, is there one planned? Personally, I have the tools to create an installation package that copies different files to different locations in Joomla, but I'd prefer some standard procedure - a uniform installation format.

Thank you, Jan

**Comment by dgrammatiko** [comment]
> let's say I have created a child template and now I would like to distribute it to users.

Child templates are NOT supposed to become distributed. They were introduced to solve the lack of persisting user edited parts of the template...

**Reply by PhocaCz** [reply]
Yes, I understand, but people have been asking me for a copy of the Cassiopeia child template I've created, and I'm interested in making it as easy as possible for them to create such a child template on their site (and the easiest solution is always a simple import using the installer package).

**Reply by dgrammatiko** [reply]
Child templates are ACTUAL templates, zip all the files (media folder included) and install it as a normal template. There's nothing more to it. Just make it clear that this is USER owned code, not yours (as a DEV), ie no updates, etc...
